addons/sale_timesheet/tests/test_sale_timesheet.py

```diff
@@ -1089,6 +1089,37 @@ def test_timesheet_get_accounts_from_sol(self):
         self.assertFalse(timesheet[other_analytic_plan2._column_name()])
         self.assertEqual(timesheet[other_analytic_plan3._column_name()], analytic_account3)
 
+    def test_timesheet_get_accounts_from_sol_fallback_on_project(self):
+        _project_analytic_plan, other_plans = self.env['account.analytic.plan']._get_all_plans()
```

Comment:
looks a bit strange given that ``_project_analytic_plan`` is never used :thinking: can't we use an index in the get_all_plans() method or perhaps rename to something like ``_dummy`` so people don't take a second look at it?

Reply:
It seems it's a convention we are using at some places in the code, I would propose to stay consistent (e.g. https://github.com/odoo/odoo/blob/18.0/addons/sale_timesheet/tests/common.py#L59)

Reply:
I usually use `_dummy`, don't think it matters tho